hipparchus-optim/src/main/java/org/hipparchus/optim/nonlinear/vector/leastsquares/SequentialGaussNewtonOptimizer.java

```diff
@@ -0,0 +1,340 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
For new files, the header should refer to Hipparchus, not Apache.
See for example the files added recently in org.hipparchus.special.jacobi